testbdd/README.md

```diff
@@ -0,0 +1,20 @@
+# BDD tests
+
+Tests in this module are a rewrite and enhancement of the end-to-end tests available in the `/test` directory.
+They use [Godog](https://github.com/cucumber/godog) framework which is the official [Cucumber](https://cucumber.io/) BDD framework for Go and use [Gherkin](https://cucumber.io/docs/gherkin/) for writing test scenarios.
+
+Tests also make use of the [kogito-operator](https://github.com/kiegroup/kogito-operator/tree/main/test) framework for Kubernetes which simplifies communication with a Kubernetes cluster.
```

Comment:
Can't we bring it here? Not sure if I like the idea of importing this module.

Reply:
you should extract the bdd test framework in a separate repository. and reuse in both operator repositories